cfgov/unprocessed/js/routes/on-demand/simple-chart/data-filters.js

```diff
@@ -46,7 +46,7 @@ function makeSelectFilterDOM(options, chartNode, filter) {
   )[0];
 
   const wrapper = document.createElement('div');
-  wrapper.className = 'filter-wrapper m-form-field m-form-field__select';
+  wrapper.className = 'filter-wrapper m-form-field';
```

Comment:
This change could be scrutinized. It didn't look like `m-form-field__select` had any special use here, but please double-check. It doesn't look like this code is actually used in production.

Reply:
Yep, no problem here.